src/domains/nodo-secret/storageaccount.tf

```diff
@@ -1,19 +1,19 @@
 module "nodocerts_sa" {
-  source = "git::https://github.com/pagopa/azurerm.git//storage_account?ref=v2.0.28"
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//storage_account?ref=v6.4.1"
 
-  name                       = replace("${local.product}-${var.domain}-nodocerts-sa", "-", "") # nodocerts<dev|uat|prod>
-  account_kind               = "StorageV2"
-  account_tier               = "Standard"
-  account_replication_type   = "LRS"
-  access_tier                = "Hot"
-  versioning_name            = "versioning"
-  enable_versioning          = false
-  resource_group_name        = azurerm_resource_group.sec_rg.name
-  location                   = var.location
-  advanced_threat_protection = false
-  allow_blob_public_access   = false
+  name                     = replace("${local.product}-${var.domain}-nodocerts-sa", "-", "") # nodocerts<dev|uat|prod>
+  account_kind             = "StorageV2"
+  account_tier             = "Standard"
+  account_replication_type = "LRS"
```

Comment:
is correct that must be "LRS" and not GRS o something more?

Reply:
at the moment is correct. with DR implementation will be migrate to GZRS